### DISCUSSION TITLE
Proposal submission

### DISCUSSION THREAD

**Discussion by KSSSenapati** [github/india#640] [Students]
Can anyone guide where exactly do we need to post/upload our proposal for externship program??

**Comment by punitjain03** [comment]
Once the projects are live, You will get to know the proposal methodologies to apply for your dream organisations

**Reply by SAIREDDY54** [reply]
@punitjain03 can you please share the proposal format available if any.

**Reply by chaudharyv714** [reply]
Yes the format or  previous year proposals would be really handy for reference.
